receiver/vcenterreceiver/metrics.go

```diff
@@ -453,7 +453,8 @@ func (v *vcenterMetricScraper) recordHostVSANMetrics(vSANMetrics *VSANMetricResu
 			case "latencyWrite":
 				v.mb.RecordVcenterHostVsanLatencyAvgDataPoint(pcommon.NewTimestampFromTime(*timestamp), value, metadata.AttributeVsanLatencyTypeWrite)
 			case "congestion":
-				v.mb.RecordVcenterHostVsanCongestionDataPoint(pcommon.NewTimestampFromTime(*timestamp), value)
+				congestionRate := float64(value) / float64(metric.Interval)
+				v.mb.RecordVcenterHostVsanCongestionDataPoint(pcommon.NewTimestampFromTime(*timestamp), int64(congestionRate))
```

Comment:
Shouldn't be converting congestionRate to an int64. it should be a float in the metadata.yaml

Reply:
resolve